appendices/VK_EXT_acquire_drm_display.txt

```diff
@@ -0,0 +1,28 @@
+// Copyright 2018-2021 The Khronos Group, Inc.
+//
+// SPDX-License-Identifier: CC-BY-4.0
+
+*Last Modified Date*::
+    2021-05-11
+*IP Status*::
+    No known IP claims.
+*Contributors*::
+  - Simon Zeni, Status Holdings, Ltd.
+
+=== Description
+
+This extension allows an application to take exclusive control of a display
+using the Direct Rendering Manager (DRM) interface. When acquired, the display
+will be under full control of the application until it is released or the
+connection is closed.
```

Comment:
What connection?

Reply:
```diff
 using the Direct Rendering Manager (DRM) interface. When acquired, the display
-will be under full control of the application until it is released or the
-connection is closed.
+will be under full control of the application until the display is either
+released, or the display connection is closed or lost.
```

Should it be more specific about the connection being the physical cable?

Reply:
If this is trying to deal with hot-unplug, it might be better to write some text detailing interactions between leases and unplug events and then reference that here. I think the display_control extension has some hotplug functionality.

I was concerned this language was left over from spec language dealing with wl_display connections.

Reply:
Closing the connection refers mainly to closing the DRM fd (either master or leased). Lost would be more about pulling the cable from the display connector or a power loss.

Reply:
I don't know that this is sufficiently clear to readers then. It wasn't to me at least. I would prefer it be clarified (use text closer to what you've used above to explain what connection closed means), but up to you.